packages/utils/src/formatDataValueByType.js

```diff
@@ -81,8 +81,8 @@ const percentage = value => {
   return `${Math.round(percentageValue * floatNormalizer) / floatNormalizer}%`;
 };
 
-const number = (value, { presentationOptions = {} }) => {
-  const { valueFormat = '0,0' } = presentationOptions;
+const number = (value, { presentationOptions }) => {
+  const valueFormat = presentationOptions?.dataValueFormat ? valueFormat : '0,0';
```

Comment:
Unless I'm missing something, `valueFormat` isn't defined yet?
```suggestion
  const valueFormat = presentationOptions?.dataValueFormat || '0,0';
```

Reply:
woops you are totally right. I've fixed it now.